media_player/alexa.py

```diff
@@ -12,6 +12,9 @@
 import requests
 import voluptuous as vol
 from bs4 import BeautifulSoup
+import pickle
+from threading import Lock
```

Comment:
This thread locking will not pass the requirements to submit to HA as an official component. We need to avoid locking/blocking the component thread at all costs.

Reply:
@keatontaylor The lock was probably unnecessary but came from the other ha component is lifted the code from. That was a sensor so a lock was probably critical on the file access. Should be fine to remove it.

Reply:
Removed.